### ISSUE TITLE
About the Operation of the Newly Added Visualize

### ISSUE THREAD

**Issue by Zengyf-CVer** [ultralytics/yolov5#3924]
@glenn-jocher 

I just ran the newly added feature map visualization command:
```shell
python detect.py --visualize --weights ./weight/yolov5s.pt --source ./train2017/000000000081.jpg
```
Then this error was reported in the shell:
```
TypeError: can't convert cuda:0 device type tensor to numpy. Use Tensor.cpu() to copy the tensor to host memory first.
```
The following is a specific screenshot:
![ksnip_20210707-221125](https://user-images.githubusercontent.com/41098760/124774354-53314080-df70-11eb-9c13-53252845e062.png)

My configuration:
- python 3.8
- CUDA 11.1
- PyTorch 1.9.0

**Comment by glenn-jocher** [comment]
@Zengyf-CVer ah, thanks for the bug report.

**Comment by glenn-jocher** [comment]
@Zengyf-CVer good news 😃! Your original issue may now be fixed ✅ in PR #3925. To receive this update:
- **[Git](https://github.com/ultralytics/yolov5)** – `git pull` from within your `yolov5/` directory or `git clone https://github.com/ultralytics/yolov5` again
- **[PyTorch Hub](https://pytorch.org/hub/ultralytics_yolov5/)** – Force-reload with `model = torch.hub.load('ultralytics/yolov5', 'yolov5s', force_reload=True)`
- **[Notebooks](https://github.com/ultralytics/yolov5/blob/master/tutorial.ipynb)** – View updated notebooks  <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **[Docker](https://hub.docker.com/r/ultralytics/yolov5)** – `sudo docker pull ultralytics/yolov5:latest` to update your image <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>

Thank you for spotting this issue and informing us of the problem. Please let us know if this update resolves the issue for you, and feel free to inform us of any other **issues** you discover or **feature requests** that come to mind. Happy trainings with YOLOv5 🚀!
